Packs/NucleonCyber/Integrations/NucleonCyberFeed/NucleonCyberFeed.py

```diff
@@ -0,0 +1,761 @@
+        )
+
+        json_payload = json.loads(res)
+        result = []
+        # all_data = json_payload.get("data")
+        all_urls = json_payload.get("data").get("hash")
+        for data in all_urls:
+            result.append(
+                {
+                    'value': data,
+                    "type": "File",
+                    # "reputation":"Bad",
+                }
+            )
+        return result
+
+    """
+    """
+
+    def get_urls(self, limit: int) -> List:
+        res = self._http_request('GET',
+                                 url_suffix='',
+                                 full_url="http://api.cybercure.ai/feed/get_url",
+                                 resp_type='text',
+                                 )
+
+        json_payload = json.loads(res)
+        result = []
+        # all_data = json_payload.get("data")
+        all_urls = json_payload.get("data").get("urls")
+        for data in all_urls:
+            result.append(
+                {
+                    'value': data,
+                    "type": "URL",
+                    # "reputation":"Bad",
+
+                }
+            )
+        return result
+
+    """
+    """
+
+    def fix_segment(self, segment) -> str:
+        if segment is None:
+            return "other"
+        else:
+            return segment
+
+    """
+    """
+
+    def get_ips(self, params: Dict[str, str], limit: int) -> List:
+        global_username = params.get('username')
+        global_password = params.get('password')
+        global_usrn = params.get('usrn')
+        global_client_id = params.get('clientid')
+        # ips_indicators_url = params.get('ips')
+        body = {'usrn': global_usrn, 'clientID': global_client_id, 'limit': limit}
+        res = self._http_request('POST',
+                                 url_suffix='',
+                                 full_url="https://api.nucleoncyber.com/feed/activethreats",
+                                 auth=(global_username, global_password),
+                                 data=body,
+                                 resp_type='text',
+                                 )
+        json_payload = json.loads(res)
+        result = []
+        all_data = json_payload.get("data")
+
+        demisto.info(all_data)
+        for data in all_data:
+            result.append(
+                {
+                    'value': data.get("ip"),
+                    "exp": data.get("exp"),
+                    'type': "IP",
+                    'segment': self.fix_segment(data.get("attackDetails").get("segment")),
+                    'targetCountry': data.get("attackDetails").get("targetCountry"),
+                    'os': data.get("attackDetails").get("remote").get("os"),
+                    'osVersion': data.get("attackDetails").get("remote").get("osVersion"),
+                    'governments': data.get("attackMeta").get("governments"),
+                    'port': data.get("attackMeta").get("port"),
```

Comment:
Could there be a case where "attackMeta" will not exist in the response?

Reply:
validated if attackMeta, remote and attackDetails exists before this assignment (if not it will continue to the next indicator).